src/javascript/app/pages/user/account/settings/professional_client.js

```diff
@@ -18,6 +18,12 @@ const professionalClient = (() => {
         populateProfessionalClient(is_financial);
     };
 
+    const setVisible = (set_visible) => {
```

Comment:
perhaps renaming to `selector` would be better, set_visible sounds like a boolean, not a selector

Reply:
You are right